iceoryx_examples/iceperf/CMakeLists.txt

```diff
@@ -4,12 +4,25 @@ project(iceperf)
 
 find_package(iceoryx_posh CONFIG REQUIRED)
 
+get_target_property(ICEORYX_CXX_STANDARD iceoryx_posh::iceoryx_posh CXX_STANDARD)
+
 add_executable(iceperf-laurel iceperf_laurel.cpp)
+set_target_properties(iceperf-laurel PROPERTIES 
+    CXX_STANDARD_REQUIRED ON
+    CXX_STANDARD ${ICEORYX_CXX_STANDARD}
+    POSITION_INDEPENDENT_CODE ON
```

Comment:
Is this the replacement for the fPic flag?

Reply:
yes it is - it is also much cleaner since it is compiler independent